client/ts/src/client.ts

```diff
@@ -478,6 +555,67 @@ export class ManifestClient {
       },
     );
   }
+
+  /**
+   * CancelAll instruction. Cancels all orders on a market
+   *
+   * @returns TransactionInstruction
+   */
+  public cancelAllIx(): TransactionInstruction {
+    return createBatchUpdateInstruction(
+      {
+        payer: this.payer,
+        market: this.market.address,
+        manifestProgram: MANIFEST_PROGRAM_ID,
+        owner: this.payer,
+        wrapperState: this.wrapper.address,
+      },
+      {
+        params: {
+          cancels: [],
+          cancelAll: true,
+          orders: [],
+          traderIndexHint: null,
+        },
+      },
+    );
+  }
+
+  /**
+   * ClearOutMarketTxs transactions. Pulls all orders
```

Comment:
Generally, a function name that ends in
ix or ixs   returns instructions
tx or txs  returns transcations
neither    does the action

so I would name this just
clearOutMarket